client/dashboard/sites/site-status/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ExternalLink } from '@wordpress/components';
+import { useAnalytics } from '../../app/analytics';
+import ComponentViewTracker from '../../components/component-view-tracker';
+import { getSiteStatusLabel } from '../../utils/site-status';
+import type { Site } from '../../data/types';
+
+function ComingSoonStatusLink( {
+	href,
+	children,
+	style,
+}: {
+	href: string;
+	children: React.ReactNode;
+	style?: React.CSSProperties;
+} ) {
+	const { recordTracksEvent } = useAnalytics();
+
+	return (
+		<>
+			<ComponentViewTracker eventName="calypso_dashboard_site_launch_nag_impression" />
+			<ExternalLink
+				style={ style }
+				href={ href }
+				onClick={ () => {
+					recordTracksEvent( 'calypso_dashboard_site_launch_nag_click' );
+				} }
+			>
+				{ children }
+			</ExternalLink>
+		</>
+	);
+}
+
+export default function SiteStatus( { site, style }: { site: Site; style?: React.CSSProperties } ) {
+	const label = getSiteStatusLabel( site );
+	if ( site.launch_status !== 'unlaunched' ) {
+		return <span style={ style }>{ label }</span>;
```

Comment:
I think we should just render the label. The `<span>` is giving more height than the rest of the fields, which make it look unbalanced.

```suggestion
		return label;
```

Reply:
Yep, seems like the `display: inline-block` style that's being passed here is giving some extra height. I don't think we need those overrides if the status is not a link.